### ISSUE TITLE
Remove GitHub Action `set-output` Commands Before May 2023

### ISSUE THREAD

**Issue by axisofentropy** [UffizziCloud/uffizzi_cli#174]
GitHub's `set-output` command is deprecated and so we must stop using it before May 2023. Even now its use generates warnings within workflows that are undesirable, especially for our demonstrations. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Instead of writing `set-output` commands to `STDOUT`, our CLI should instead open and **append** to the environment file provided by GitHub. The filename is the value of the environment variable `GITHUB_ENV`.

I'd also like to automatically detect when our CLI is invoked within a GitHub Action.  The `GITHUB_ACTIONS` environment variable can be used for this; its value is always `true` within a GitHub Action runner environment.

This will require coordination with the `preview-action` repository to use the new environment variables specified by this new feature.

Leave the existing `--output=github-action` behavior unchanged for backwards compatibility but add comments to remove this after May 2023.

**Comment by gadkins** [comment]
Implemented here https://github.com/UffizziCloud/preview-action/pull/35

**Comment by axisofentropy** [comment]
I do not think this was implemented. This ticket was for the CLI itself. It's still outputting as seen here: https://github.com/nocodb/nocodb/actions/runs/3623455656/jobs/6109341931#step:16:21
![image](https://user-images.githubusercontent.com/66315/205748135-4917fb6e-d13f-47b1-a6a1-3d6817854ceb.png)

**Comment by axisofentropy** [comment]
> ## Test Cases

Instead of writing to `GITHUB_ENV`, it's probably better to use `GITHUB_OUTPUT`, which may not have been available last October.

**Comment by moklidia** [comment]
https://github.com/UffizziCloud/uffizzi_cli/issues/214 implemented in the same branch

**Comment by moklidia** [comment]
Open a new PR:
<details>

<img width="1326" alt="Снимок экрана 2023-02-21 в 13 03 40" src="https://user-images.githubusercontent.com/42300552/220339955-9738214e-45c3-434d-b44d-1272d86e1345.png">

</details>

Update a preview:

<details>
<img width="1170" alt="Снимок экрана 2023-02-21 в 13 10 44" src="https://user-images.githubusercontent.com/42300552/220341380-f08613df-377a-4b74-9c3f-a5bdd709325a.png">

</details>

**Comment by NealArw** [comment]
**TESTED - OK**

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/220612392-869112d7-b1d6-414c-9b0b-f80acc1d2088.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/220612454-037fa4cb-6380-4d18-a4ba-012a86a3a590.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/220615630-71f9ca76-d8af-41db-99c6-f771e91174cb.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/220640877-ca4098dd-b844-4f80-af3f-613d2af8ada2.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/220640960-e061138a-196b-4eb1-acd7-464ff710ea5b.png">

**Comment by NealArw** [comment]
TESTED on prod - 

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/221592688-5ecff5a4-614a-4c1f-944b-ac21b12d5d9f.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/221601965-d7bacd9e-a6c8-42da-bdbe-d58c2f3df9c5.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/221607745-915b22d3-745b-4af9-9638-d0bcfda3d726.png">

![github com_NealArw_quickstart_actions_runs_4283746640 (1)](https://user-images.githubusercontent.com/89470900/221602390-4c18cb89-e698-4d1e-a1e4-da933ba722ef.png)

![github com_NealArw_quickstart_actions_runs_4283746640](https://user-images.githubusercontent.com/89470900/221602398-16829094-b12c-4604-be74-32781e4ff971.png)


<img width="1680" alt="image" src="https://user-images.githubusercontent.com/89470900/221599142-5a7eadac-bb4c-4fc6-8760-806233338bce.png">
